src/navigation.js

```diff
@@ -0,0 +1,47 @@
+// sticky menu
+
+const header = document.getElementById('header');
+const sticky = header.offsetTop;
+
+function stickMenu() {
+  if (window.pageYOffset > sticky) {
+    header.classList.add('sticky');
+  } else {
+    header.classList.remove('sticky');
+  }
+}
+
+window.onscroll = () => { stickMenu(); };
+
+// nav bar
+
+const showBookList = document.querySelector('#showList');
+const showNewAdd = document.querySelector('#showAddBook');
+const showContact = document.querySelector('#showContact');
+
+const bookContainer = document.querySelector('#book-container');
+const addNew = document.querySelector('#add-new');
+const contactSection = document.querySelector('#contact-section');
+
+addNew.classList.add('hideSection');
+contactSection.classList.add('hideSection');
+
+function showPage(section) {
```

Comment:
- [ ] OPTIONAL
It is nice to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.